src/components/Matching/MatchSvg/ImageChoice.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <image class="image-choice" :href="url" />
+  <image class="image-choice" :href="url"/>
```

Comment:
lets leave whitespace changes alone unless we're doing a pass in a separate "cleaning" type pull request